### PR TITLE
Makes fire blossoms glowy.

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
They have bioluminescence trait as endershrooms, but don't glow like endershrooms.
That's how it looks when grown:
image
And it has a weaker light when it's harvested:
image
## Why It's Good For The Game
It's cool.
## Changelog
:cl:
add: Fire Blossoms are now glowy.
/:cl:

### CODE REVIEW

File: code/modules/mining/lavaland/ash_flora.dm [hunk 1]
```diff
@@ -133,6 +133,9 @@
 	desc = "An odd flower that grows commonly near bodies of lava."
 	icon_state = "fireblossom1"
 	base_icon_state = "fireblossom"
+	light_range = 2.1
+	light_power = 2.1
+	light_color = "#FFFF66"
```
Comment (Mothblocks) [inline code/modules/mining/lavaland/ash_flora.dm:138]: Should use color defines

File: code/modules/mining/lavaland/ash_flora.dm [hunk 2]
```diff
@@ -144,6 +147,18 @@
 	regrowth_time_high = 4000
 	number_of_variants = 2
 
+/obj/structure/flora/ash/fireblossom/after_harvest()
+	set_light_power(2.7)
+	set_light_range(1.5)
+	update_light()
+	. = ..()
```
Comment (Mothblocks) [inline code/modules/mining/lavaland/ash_flora.dm:154]:
```suggestion
	return ..()
```

File: code/modules/mining/lavaland/ash_flora.dm [hunk 3]
```diff
@@ -144,6 +147,18 @@
...
+
+/obj/structure/flora/ash/fireblossom/regrow()
+	set_light_power(initial(light_power))
+	set_light_range(initial(light_range))
+	update_light()
+	. = ..()
```
Comment (Mothblocks) [inline code/modules/mining/lavaland/ash_flora.dm:160]:
```suggestion
	return ..()
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by MrMelbert** [comment]
Would be cool if this was tied to the gene rather than hardcoded. If you can't figure out a way to do that I don't mind, but just an idea

**Comment by Helg2** [comment]
> Would be cool if this was tied to the gene rather than hardcoded. If you can't figure out a way to do that I don't mind, but just an idea

No idea tbh

File: code/modules/mining/lavaland/ash_flora.dm [hunk 4]
```diff
@@ -144,6 +147,18 @@
 	regrowth_time_high = 4000
 	number_of_variants = 2
 
+/obj/structure/flora/ash/fireblossom/after_harvest()
+	set_light_power(2.7)
```
[the remark this answers is not in the document]
Comment (Helg2) [reply code/modules/mining/lavaland/ash_flora.dm]: So, uh, something like this?

[the remark this answers is not in the document]
Comment (Helg2) [reply code/modules/mining/lavaland/ash_flora.dm]: so, something like this?

**Review by ZephyrTFA — APPROVED** [review]

File: code/modules/mining/lavaland/ash_flora.dm (hunk 4 above)
[the remark this answers is not in the document]
Comment (ZephyrTFA) [reply code/modules/mining/lavaland/ash_flora.dm]: aye

**Pull request merged**